lib/util.js

```diff
@@ -298,11 +298,13 @@ module.exports = {
         }
 
         // split the path into indiv. segments for trie generation
-        currentPath = path.split('/').filter((pathItem) => {
+        // unless path it the root endpoint
+        currentPath = path === '' ? [''] : path.split('/').filter((pathItem) => {
```

Comment:
@pitpit Can this be changed to `path === '' ? ['(root)'] : path.split('/').filter((pathItem) => {`, or similar? An emppy string creates a folder with no name in case two requests are present under the `/` path:

![Screen Shot 2019-07-05 at 11 08 08](https://user-images.githubusercontent.com/681190/60700343-30996680-9f15-11e9-9847-8bd963d39959.png)

Reply:
fixed